health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UserAccess.js

```
@@ -167,7 +192,23 @@ function UserAccess({ category, setData, nationalRoles }) {
+                        maxWidth: "180px",
+                        whiteSpace: "normal",
+                        height: "100%",
+                      },
+                      textStyles:{
+                        whiteSpace:'normal'
+                      }
+                    }}
+                    iconReq=""
+                    hideClose={true}
+                    text={t(item)}
+                  />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)